xml/System.Xml/XmlDocumentXPathExtensions.xml

```diff
@@ -41,9 +41,9 @@
         <Parameter Name="document" Type="System.Xml.XmlDocument" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="document">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="document">Document to create XPath navigator from.</param>
+        <summary>Creates a new <see cref="T:System.Xml.XPath.XPathNavigator" /> object for navigating this document.</summary>
```

Comment:
Does this overload position the navigator at the root? I ask because there is an overload below that takes a node as well, so if this one does not take a node, I would assume the root is where this navigator would begin. If yes, maybe we should mention that.

Reply:
yes, I copied most of these description from the original type (see i.e. XmlDocument.CreateNavigator)